File: tools/mk-cmd-list-of-fc.pl
#!/usr/bin/perl

use strict;

while (<>) {
  if (m@Selenium\.prototype\.(do|get)(\w+)\s+=\s+@) {
    if ($1 eq 'do') {
      print lcfirst($2), "\n";
    } elsif ($1 eq 'get') {
      print "store$2\n";
    }
  }
}

File: tools/mk-cmd-list-of-fc.rb
#!/usr/bin/ruby
# -*- coding: utf-8 -*-

ARGF.each_line do |line|
  if /Selenium\.prototype\.(do|get)(\w+)\s+=\s+/ =~ line
    name = $2
    if $1 == 'do'
      name[0] = name[0].downcase
    elsif $1 =='get'
      name = "store" + name
    end
    puts name
  end
end
